.circleci/deployment/commands.yml

```diff
@@ -97,6 +97,10 @@
             sudo apt update
             sudo add-apt-repository ppa:rmescandon/yq
             sudo apt-get install yq
+      - run:
```

Comment:
@jtimpe noticed [this](https://app.circleci.com/pipelines/github/raft-tech/TANF-app/20691/workflows/6d6b2c60-ec5f-4932-ac55-c20249c34f44/jobs/61537?invite=true#step-107-27674_93) -- is this okay?

Reply:
nvm. i see it referenced below :)

Reply:
i believe this is related to the `graphviz` dependency, andrew and i discussed that a bit in [this comment](https://github.com/raft-tech/TANF-app/pull/2779#discussion_r1499802211). long story (not so) short, compilation of `pygraphviz` fails because we eliminated the `graphviz` os dependency. it's not needed to run the migrations so should not cause an issue, but will be present in every run. the script does not stop on account of that error (or any dependency compilation errors, for that matter), but if anything blocks execution of `python manage.py migration`, the job will fail and the deployment will quit